gamearea.py

```diff
@@ -501,3 +554,26 @@ def start_timeout(self, time, callback=None, force=False):
 
     def redraw(self):
         GObject.idle_add(self.queue_draw)
+    
+    def save_highscore(self):
+        file_path = os.path.join(get_activity_root(), 'data', 'highscore')
+        highscore = [0]
+        if os.path.exists(file_path):
+            with open(file_path, "r") as fp:
+                highscore = fp.readlines()
+
+        int_highscore = int(highscore[0])
+        if not int_highscore > self.score:
+            with open(file_path, "w") as fp:
+                fp.write(str(self.score))
+
+    def load_highscore(self):
+        file_path = os.path.join(get_activity_root(), 'data', 'highscore')
+        if os.path.exists(file_path):
+            try:
+                with open(file_path, "r") as fp:
+                    highscore = fp.readlines()
```

Comment:
This can go outside the `try` block imo

Reply:
yeah, that could work too, I'll change it